pkg/controller/petset/pet.go

```diff
@@ -251,8 +251,8 @@ func (p *apiServerPetClient) SyncPVCs(pet *pcb) error {
 		if !exists {
 			if err := p.createPVC(&pet.pvcs[i]); err != nil {
 				errMsg += fmt.Sprintf("Failed to create %v: %v", pvc.Name, err)
+				p.event(pet.parent, "Create", fmt.Sprintf("pvc: %v", pvc.Name), err)
 			}
-			p.event(pet.parent, "Create", fmt.Sprintf("pvc: %v", pvc.Name), err)
```

Comment:
getPVC is a private method so reutrning the raw error might be ok right now, but I'd still rather not because I want to be consistent with the public Get functions of the interface which reutrn a bool for not found for the stated reason. Being consistent makes exposing getPVC easier, if we should need to.